lib/core/src/lib/datatable/components/datatable/datatable.component.html

```diff
@@ -14,26 +14,30 @@
             data-automation-id="datatable-row-header"
             [disabled]="!isHeaderVisible()"
             class="adf-datatable-row"
-            role="row">
-
+            role="row"> 
 
             <!-- Drag -->
             <div *ngIf="enableDragRows" class="adf-datatable-cell-header adf-drag-column">
-                <span class="adf-sr-only">{{ 'ADF-DATATABLE.ACCESSIBILITY.DRAG' | translate }}</span>
+                <div cdkDragHandle 
```

Comment:
This is sr only so it means that this element is added purely for a screen reader not as actual element so it should not have the `cdkDragHandle` as you can see there is already the other element that has it applied